src/api/comment.ts

```diff
@@ -0,0 +1,14 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getComments = (postId: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
+
+export function deleteComment(commentId: number) {
+  return client.delete(`/comments/${commentId}`);
+}
+
+export function addComment({ postId, name, email, body }: Comment) {
+  return client.post<Comment>('/comments', { postId, name, email, body });
+}
```

Comment:
The `addComment` function destructures its parameter as if it is an object of type `Comment`, but it should be an object containing the properties `postId`, `name`, `email`, and `body`. Ensure that the parameter passed to `addComment` matches this structure, or adjust the function signature to accept a `Comment` object directly.